Sitemap.php

```diff
@@ -202,42 +203,46 @@ private function getCurrentFilePath()
         }
 
         $parts = pathinfo($this->filePath);
-        return $parts['dirname'] . DIRECTORY_SEPARATOR . $parts['filename'] . '_' . $this->fileCount . '.' . $parts['extension'];
+
+        return $parts['dirname'].DIRECTORY_SEPARATOR.$parts['filename'].'_'.$this->fileCount.'.'.$parts['extension'];
     }
 
     /**
-     * Returns an array of URLs written
+     * Returns an array of URLs written.
      *
      * @param string $baseUrl base URL of all the sitemaps written
+     *
      * @return array URLs of sitemaps written
      */
     public function getSitemapUrls($baseUrl)
     {
         $urls = [];
         foreach ($this->writtenFilePaths as $file) {
-            $urls[] = $baseUrl . pathinfo($file, PATHINFO_BASENAME);
+            $urls[] = $baseUrl.pathinfo($file, PATHINFO_BASENAME);
         }
+
         return $urls;
     }
 
     /**
      * Sets maximum number of URLs to write in a single file.
      * Default is 50000.
-     * @param integer $number
+     *
+     * @param int $number
      */
     public function setMaxUrls($number)
     {
-        $this->maxUrls = (int)$number;
+        $this->maxUrls = (int) $number;
     }
 
     /**
      * Sets number of URLs to be kept in memory before writing it to file.
      * Default is 1000.
      *
-     * @param integer $number
+     * @param int $number
      */
     public function setBufferSize($number)
     {
-        $this->bufferSize = (int)$number;
+        $this->bufferSize = (int) $number;
```

Comment:
Actually some of the modifications are made by PHPCS fixer  (https://github.com/FriendsOfPHP/PHP-CS-Fixer) ;